src/parser/priest/holy/modules/core/EchoOfLightMastery.js

```diff
@@ -97,6 +99,10 @@ class EchoOfLightMastery extends Analyzer {
       this.targetMasteryPool[targetId].pendingHealingBySpell[spellId] = 0;
     }
     this.targetMasteryPool[targetId].pendingHealingBySpell[spellId] += rawHealing;
+    if (event.hitType === HIT_TYPES.CRIT) {
+      // This is just to track how much of a EoL tick can be contributed to a crit.
```

Comment:
Definitely great you've left a comment here, I'd make a suggestion to put it in a less passive voice, 
```suggestion
      // Track how much of a EoL tick can be contributed to a crit.
```

Google have an excellent guide to code comments and PRs in general: https://developers.google.com/style/voice